src/dynamic_honey_badger/sender_queue.rs

```diff
@@ -0,0 +1,48 @@
+use std::cmp;
+
+use rand::Rand;
+use serde::{de::DeserializeOwned, Serialize};
+
+use super::{Batch, Change, ChangeState, DynamicHoneyBadger, Epoch, Message, Step};
+use sender_queue::SenderQueueFunc;
+use {Contribution, Epoched, NodeIdT};
+
+impl<C, N> SenderQueueFunc<DynamicHoneyBadger<C, N>> for DynamicHoneyBadger<C, N>
```

Comment:
I wonder whether the trait should be called something like `Queueable` or `SenderQueueable`?
In any case, I'd make the file name the same as the trait's.